models/setting.js

```diff
@@ -3,8 +3,7 @@ const Schema = mongoose.Schema;
 
 const SettingSchema = new Schema({
   id: {type: String, default: '1'},
```

Comment:
I thought we'd discussed hardcoding it to `"1"` since there is only ever one record in this collection, and there's an init file I added to `/bin` that creates the record with an id of `1`